src/tools/generate/generate_syntax_highlighting.js

```diff
@@ -0,0 +1,43 @@
+const { JSDOM } = require('jsdom');
+const rainbow = require('rainbow-code');
+
+
+/**
+ * Generate Syntax highlighting for a particular Languaage (ex: html, css, javascript etc..)
+ *
+ * @param {object} dom Object of class JSDOM (Parsed HTML).
+ * @param {String} body HTML body parsed from convertor
+ * @param {String} language language which rainbow syntax highlighting to use
+ * @param {String} [alias] an optional alias for a language (e.g. js for javascript)
+ * @returns body with highlighting syntax snippet of a particular language.
+ */
+const generate_syntax_highlighting_for_language = (dom, body, language, alias="") => {
+  const query_selector = alias ? `code.language-${alias}` : `code.language-${language}`;
```

Comment:
My suggestion for this line was actually a tiny bit different:

```suggestion
  const query_selector = alias ? `code.language-${language}, code.language-${alias}` : `code.language-${language}`
```

That way it will do both `javascript` and `js` in the one go!

Reply:
Okay, yeah make sense